WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/BlogDashboardViewModel.swift

```diff
@@ -225,3 +225,9 @@ private extension Collection where Element == DashboardCardModel {
         return contains(where: { $0.cardType == .scheduledPosts })
     }
 }
+
+extension Collection where Element == DashboardCardModel {
```

Comment:
Can we keep this one private to avoid unintented O(n) access. I know in our world these things often don't matter but it would be a very easy improvement imo.

Reply:
If I did that then it wouldn't be accessible here: https://github.com/wordpress-mobile/WordPress-iOS/blob/8a8081edc8b602c770c3866ae9bfa533bda4abc1/WordPress/Classes/ViewRelated/Blog/Blog%20Dashboard/BlogDashboardViewController.swift#L77

I could just remove the `hasPrompts` call and replace it with the `contains` call in the `guard` if you think that's better. I mainly added it to match the existing code already there: https://github.com/wordpress-mobile/WordPress-iOS/blob/8a8081edc8b602c770c3866ae9bfa533bda4abc1/WordPress/Classes/ViewRelated/Blog/Blog%20Dashboard/ViewModel/BlogDashboardViewModel.swift#L220-L226

Reply:
> I could move the extension into BlogDashboardViewController

I ended up just doing this in: https://github.com/wordpress-mobile/WordPress-iOS/pull/19846/commits/99b699eeedac6acb2195c20c95246751aec8a0a8